docs/resume.rst

```diff
@@ -7,12 +7,12 @@ An important aspect of pgcopydb design is detailed in the documentation
 section :ref:`pgcopydb_concurrency` and has to do with using many concurrent
 worker processes to implement parallelism.
 
-Even when using multiple worker processes, it is important that pgcopydb
-operations are consistent. It is essential to guarantee that the same source
-schema and data set is used by every single worker process throughout the
-operations.
+When using multiple worker processes, it is important that pgcopydb
+operations produce consistent output. It is essential to guarantee that 
+the same source schema and data set is used by every single worker process 
+throughout the operations lifecycle.
```

Comment:
I think that change is wrong. This part is not about pgcopydb “output”, it's about pgcopydb operations such as Postgres catalog queries and pg_dump and large-objects etc.

Reply:
Oh ok, sure. Reverted with a small grammar correction.